Controller/GameHandling.cs

```diff
@@ -0,0 +1,87 @@
+﻿namespace GameOfLife
+{
+    public class GameHandling
+    {
+        private bool gameIsRunning = true;
+        Game? game;
+        GameGraphic? gameGraphic;
+
+        /// <summary>
+        /// Starts and runs custom game.
+        /// </summary>
+        public void RunCustomGame()
+        {
+            int x = UserInput.GetUserInput("Enter the Heigth");
+            int y = UserInput.GetUserInput("Enter the Width");
+
+            if ((x > UserInput.maxInputValue || x < UserInput.minInputValue) || (y > UserInput.maxInputValue || y < UserInput.minInputValue))
+            {
+                Console.WriteLine(UserInput.wrongUserInput);
+            }
+            else
+            {
+                game = new Game(x, y);
+                gameGraphic = new GameGraphic(game);
+
+                while (Console.KeyAvailable == false)
+                {
+                    Console.WriteLine("Live cells: " + gameGraphic.CalculateLiveCellsCurrentGeneration() + "Iteration: " + game.iterationCount);
+                    gameGraphic.DrawGameWindow();
+                    game.GenerateNextGeneration();
+                    Thread.Sleep(1000);
+                    Console.WriteLine();
```

Comment:
```
                gameGraphic = new GameGraphic(game);

                while (Console.KeyAvailable == false)
                {
                    Console.WriteLine("Live cells: " + gameGraphic.CalculateLiveCellsCurrentGeneration() + "Iteration: " + game.iterationCount);
                    gameGraphic.DrawGameWindow();
                    game.GenerateNextGeneration();
                    Thread.Sleep(1000);
                    Console.WriteLine();
                }
````
You have this code repeated twice in RunCustomGame and RunRandomGame. You can put it in new method something like RunGame with input parameter game.

Reply:
private void UpdateGameField() method created. Method updates the game field every second.